REQUIRE

```diff
@@ -1,5 +1,5 @@
-julia 0.6
-LightGraphs 0.12
-JuMP 0.13
-MatrixDepot
-BlossomV 0.3
+julia 0.7
+LightGraphs 1.2.0
+JuMP 0.18.3
+MathProgBase 0.7.7
```

Comment:
```suggestion
MathProgBase 0.7
```
instead, no need to specify the patch

Reply:
I will change that. Is that some kind convention that we don't specify the patch version? Usually I just select the latest version, as otherwise I would have to go back and test it with all older versions.

Reply:
it avoids being dependent on a patch, since we're using the features of the exposed API, we're not dependent on patch number.
We don't need to update any code on our side if a patch happens in a dependency, the REQUIRE specifies that by omitting the patch number

Reply:
Same thing for all other dependencies we can remove patches